app/assets/stylesheets/styles/_presets.css.scss

```diff
@@ -64,6 +66,10 @@
 
 */
 
+html, body, #content {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line